src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -58,9 +60,20 @@ export default class CoveredEResourcesList extends React.Component {
     platform: e => get(e._object, 'pti.platform.name', '-'),
     package: e => get(e._object, 'pkg.name', '-'),
     coverage: e => <CoverageStatements statements={e.coverage} />,
-    isCustomCoverage: e => (e.customCoverage ? <CustomCoverageIcon /> : ''),
     accessStart: e => this.renderDate(get(e._object, 'accessStart')),
     accessEnd: e => this.renderDate(get(e._object, 'accessEnd')),
+    isCustomCoverage: line => {
+      if (!line.customCoverage) return '';
+      return (
+        <Tooltip
+          text={<FormattedMessage id="ui-agreements.customcoverages.tooltip" />}
```

Comment:
We prefer to alphasort our props in ERM modules. Ie, `id` before `text`. Just makes it easier to find props when dealing with components that have a bunch of them.